### DISCUSSION TITLE
Cycling through floating and "normal" windows at the same time.

### DISCUSSION THREAD

**Discussion by yohannd1** [hyprwm/Hyprland#3244] [Q&A]
Hello! I'm new to Hyprland (just set it up today). I am using the `master` layout as my main one, and I have set up these keybindings to cycle through my opened windows:
```
bind = SUPER,J, layoutmsg, cyclenext
bind = SUPER,K, layoutmsg, cycleprev
```
They work just fine for the windows in the layout for the "normal" windows, but they don't work at all for floating windows. Is there a way to make them work?

**Comment by jamedranoa** [comment]
If you want to cycle at the workspace level you need to use cyclenext see https://wiki.hyprland.org/Configuring/Dispatchers/#list-of-dispatchers
```
bind = SUPER, J, cyclenext
bind = SUPER, K, cyclenext, prev
```
